module-apps/application-calllog/windows/CallLogMainWindow.cpp

```diff
@@ -109,7 +109,7 @@ namespace gui
         noCallsIcons->setEdges(RectangleEdge::None);
         noCallsIcons->setMinimumSize(MyStyle::descriptionSize, MyStyle::bottomBoxHeight);
         noCallsIcons->setMargins(Margins(MyStyle::descriptionLeftMargin, 0, 0, 0));
-        auto imgLambda = [&](const UTF8 imageName) {
+        const auto imgLambda = [&](UTF8 &&imageName) {
```

Comment:
What is the reasoning behind the change to rvalue ref (also below)? AFAIU the argument is not moved inside lambda expression so the change is irrelevant.

Reply:
You're right, changed to `const &`.